wwwroot/index.html

```diff
@@ -1,17 +1,26 @@
 <!DOCTYPE HTML>
-<html>
-
+<html lang="en">
 <head>
-	<title>zkSNACKs</title>
+	<title>zkSNACKs - Unfairly Private</title>
 	<meta http-equiv="content-type" content="text/html; charset=utf-8" />
-	<meta name="description" content="Nowadays it is all about individuals’ digital privacy, governments and organisations are
-	making steps to protect privacy in the digital world. We believe that privacy is both
-	a fundamental human right and business need that should be preserved at all times.
-	This is why we focus on your digital financial privacy." />
-	<meta name="keywords"
-		content="bitcoin wallet, protect privacy, wallet, bitcoin mining, bitcoin worth, bitcoin privacy, exchange, store, bitcoin, privacy policy, financial privacy, blockchain, transparent, opensource, crypto, cryptocurrency, currency" />
+	<meta name="description" content="zkSNACKs was established to research the possibilities for Bitcoin privacy enhacements. It's main product is Wasabi, a Bitcoin privacy-oriented wallet." />
+	<meta name="keywords" content="bitcoin wallet, protect privacy, wallet, bitcoin mining, bitcoin worth, bitcoin privacy, exchange, store, bitcoin, privacy policy, financial privacy, blockchain, transparent, opensource, crypto, cryptocurrency, currency" />
```

Comment:
Shouldn't we add `CoinJoin` as a keyword.

Reply:
I still need to optimize meta keywords, even if in 2020 are pretty useless